src/puget/printer.clj

```diff
@@ -202,14 +207,17 @@
    (unknown-document value (.getName (class value)) repr))
   ([value tag repr]
    (illegal-when-strict! value)
-   [:span
-    (color-doc :class-delimiter "#<")
-    (color-doc :class-name tag)
-    (color-doc :class-delimiter "@")
-    (system-id value)
-    " "
-    repr
-    (color-doc :class-delimiter ">")]))
+   (if (:print-fallback *options*)
```

Comment:
Instead of just a boolean switch here, I think it'd be better to make `:print-fallback` a keyword, with the default being the existing unknown document behavior. This could then be a `case` statement instead.